lib/spark_api/authentication/api_auth.rb

```diff
@@ -62,7 +62,7 @@ def sign_token(path, params = {}, post_data="")
       
       # Perform an HTTP request (no data)
       def request(method, path, body, options)
-        escaped_path = Addressable::URI.escape(path)
```

Comment:
It seems that Addressable is necessary for Ruby 3.0 build, @wlmcewen

Reply:
That's probably why @philreindl added it recently. As I said in our discussion, we likely need to require 'addressable' somewhere.